### PR TITLE
:zap: game cat series 5

### PR DESCRIPTION
**Pull request by blacroix**

### CODE REVIEW

File: src/pages/end.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+const EndPage = () => {
+  return (
+    <main className="pt-4 container text-center">
+      <h1 className="text-xl">Done!</h1>
```
Comment (keabard) [inline src/pages/end.tsx:6]: We need to add the score and the replay button

**Review by keabard — COMMENTED** [review]
We should take time to initialize the testing environment (unit and e2e) and a storybook if we find it useful.

**Comment by blacroix** [comment]
> We should take time to initialize the testing environment (unit and e2e) and a storybook if we find it useful.

Go 🚀

**Comment by keabard** [comment]
I'll try to catch up to your lightspeed :p

File: src/pages/end.tsx (hunk 1 above)
Comment (blacroix) [reply src/pages/end.tsx]: Done!

**Pull request merged**